flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/api/output/BoundedStreamTask.java

```diff
@@ -67,7 +67,7 @@
 	@Override
 	protected void init() throws Exception {
 		Preconditions.checkState(
-			operatorChain.getAllOperators().length == 1,
+			operatorChain.getNumberOfOperators() == 1,
```

Comment:
Reiterating our offline conversation. BoundedStreamTask uses a NeverProcessingTimeService whose timers never fire so I don't believe this will be an issue in practice. It also cannot be chained and is only used internally within the state proc api. 

https://github.com/apache/flink/blob/e1660b4b51b2a3c2fde7ff2c1f92bb0c329e602e/flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/api/output/BoundedStreamTask.java#L61